src/Tizen.Network.Stc/Tizen.Network.Stc/StcRule.cs

```diff
@@ -0,0 +1,288 @@
+        public NetworkInterface GetInterfaceType()
+        {
+            NativeNetworkInterface ifaceType;
+            int ret = Interop.Stc.Rule.GetInterfaceType(_ruleHandle, out ifaceType);
+            if (ret != (int)StcError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to get Interface type, Error - " + (StcError)ret);
+            }
+            if (ifaceType == NativeNetworkInterface.Unknown)
+            {
+                throw new InvalidOperationException("Interface Type is Unknown.");
+            }
+            return (NetworkInterface)ifaceType;
+        }
+
+        /// <summary>
+        /// To set Interface type for statistics rule.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <param name="interfaceType"> Interface type.</param>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public void SetInterfaceType(NetworkInterface interfaceType)
+        {
+            if (_disposed)
+            {
+                throw new ObjectDisposedException("Invalid StcRule instance (Object may have been disposed or released)");
+            }
+            int ret = Interop.Stc.Rule.SetInterfaceType(_ruleHandle, (NativeNetworkInterface)interfaceType);
+            if (ret != (int)StcError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to set Interface type, Error - " + (StcError)ret);
+                StcErrorFactory.ThrowStcException(ret);
+            }
+        }
+
+        /// <summary>
+        /// To get Time period for statistics rule.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <returns>Time period.</returns>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public TimePeriodType GetTimePeriod()
+        {
+            NativeTimePeriodType timePeriod;
+            int ret = Interop.Stc.Rule.GetTimePeriod(_ruleHandle, out timePeriod);
+            if (ret != (int)StcError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to get Time period, Error - " + (StcError)ret);
+            }
+            if (timePeriod == NativeTimePeriodType.Unknown)
+            {
+                throw new InvalidOperationException("Time period is Unknown.");
+            }
+            return (TimePeriodType)timePeriod;
+        }
+
+        /// <summary>
+        /// To set Time period for statistics rule.
```

Comment:
It is too confuse between `Interval` and `period`
and there is no proper description about period

Reply:
Description is added